### PR TITLE
Add more documentation-related items to definition of done

### PR DESCRIPTION
**Pull request by rumyantseva**
# Description

As discussed with @Fashander, if some user documentation needs to be written after a task implementation, it might make sense to create a separate issue about it. In this issue, we can mention the important details that need to be documented, so Alex has some input to write user docs.  

Mark as "do not merge" so everyone can take a look.

## Readiness checklist

* [ ] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [ ] I added/updated comments and checked rendering.
* [ ] I made spot refactorings.
* [ ] I updated user documentation.
* [ ] I ran `task all`, and it passed.
* [x] I ensured that PR title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/chore.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ body:
         - unit tests added/updated;
         - integration/compatibility tests added/updated;
         - spot refactorings done;
-        - user documentation updated;
+        - user documentation updated or issue to update documentation created;
```
Comment (rumyantseva) [inline .github/ISSUE_TEMPLATE/chore.yml:36]: The tricky part is that currently, we have this definition of done list for `chore` tasks only, but quite often such issues about user documentation need to be created when we implement new features. So, let me know if you see what we can do about it.

Comment (chilagrow) [reply .github/ISSUE_TEMPLATE/chore.yml:36]: I think we briefly talked that we can create a chore task, then change it to other task type.

Comment (AlekSi) [inline .github/ISSUE_TEMPLATE/chore.yml:36]:
```suggestion
        - user documentation updated or an issue to create documentation created;
```

**Review by w84thesun — APPROVED** [review]
👍

**Review by noisersup — APPROVED** [review]

**Review by AlekSi — APPROVED** [review]

**Review by chilagrow — APPROVED** [review]

**Review by Fashander — APPROVED** [review]

**Pull request merged**